crates/router/src/analytics.rs

```diff
@@ -481,21 +941,134 @@ pub mod routes {
         .await
     }
 
-    pub async fn get_frm_filters(
+    pub async fn get_payment_intents_filters(
         state: web::Data<AppState>,
         req: actix_web::HttpRequest,
-        json_payload: web::Json<GetFrmFilterRequest>,
+        json_payload: web::Json<GetPaymentIntentFiltersRequest>,
     ) -> impl Responder {
-        let flow = AnalyticsFlow::GetFrmFilters;
+        let flow = AnalyticsFlow::GetPaymentIntentFilters;
         Box::pin(api::server_wrap(
             flow,
             state,
             &req,
             json_payload.into_inner(),
-            |state, auth: AuthenticationData, req: GetFrmFilterRequest, _| async move {
-                analytics::frm::get_filters(&state.pool, req, auth.merchant_account.get_id())
-                    .await
-                    .map(ApplicationResponse::Json)
+            |state, auth: AuthenticationData, req, _| async move {
+                analytics::payment_intents::get_filters(
+                    &state.pool,
+                    req,
+                    auth.merchant_account.get_id(),
```

Comment:
Is AuthInfo not needed here?

Reply:
This API is not currently being refactored.